src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -2251,13 +2182,6 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
             ProcessEventsFor((SMART_EVENT)SMART_EVENT_TIMED_EVENT_TRIGGERED, nullptr, eventId);
             break;
         }
-        case SMART_ACTION_REMOVE_ALL_GAMEOBJECTS:
```

Comment:
Why is this one deprecated/removed?

Reply:
Because I can't imagine a scenario where creature need to remove all owned gameobjects